src/modules/commander/commander_params.c

```diff
@@ -629,6 +629,16 @@ PARAM_DEFINE_INT32(COM_ARM_MAG_ANG, 45);
  */
 PARAM_DEFINE_INT32(COM_ARM_MAG_STR, 1);
 
+/**
+ * Enable arming grace.
+ *
+ * Arming grace allows to rearm the drone with manual command without running prearmcheck during 5 s after disarming.
+ *
+ * @group Commander
+ * @boolean
+ */
+PARAM_DEFINE_INT32(COM_ARM_GRACE, 1);
```

Comment:
Not to be too pedantic, but what do you think about using rearm instead of arm everywhere?

`COM_ARM_GRACE -> COM_REARM_GRACE`

Reply:
Yes that's a bit more explicit. I will change that.